frontend/dockerfile/docs/reference.md

```diff
@@ -1472,6 +1472,10 @@ By default, the `COPY` instruction copies files from the build context. The
 COPY [--from=<stage|image>] <src> ... <dest>
 ```
 
+> **Note**
```

Comment:
This should be part of `ONBUILD` docs, not `COPY`. This is same for other commands that use `from`.